<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tome Issue Reporter</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="labelHeader">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;We're sincerely sorry you're having issues with Tome. We are.&lt;/p&gt;&lt;p&gt;You can help us to prevent that in future releases by submitting an issue report for us. Please provide as much information as possible, including what happened and what you expected to happen.&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;That issue will be publicly visible at https://github.com/npruehs/tome-editor/issues.&lt;/span&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEditTitle"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="labelTitle">
      <property name="text">
       <string>Title</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="labelDescription">
      <property name="text">
       <string>Description</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPlainTextEdit" name="plainTextEditDescription">
      <property name="backgroundVisible">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QCheckBox" name="checkBoxIncludeLog">
      <property name="text">
       <string>Upload log file to public issue tracker</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
